javascript/javascript1/week2/candy-helper.js

```diff
@@ -0,0 +1,52 @@
+const boughtCandyPrices = [];
+function addCandy(candyType, weight) {
```

Comment:
I recommend that you store the candy types and their price in a variable (all of them in one variable), and rewrite the function to avoid  using if..else statements.